pkg/storaged/utils.js

```diff
@@ -622,9 +624,13 @@ export function get_parent(client, path) {
         return client.lvols[path].VolumeGroup;
     if (client.blocks_stratis_fsys[path])
         return client.blocks_stratis_fsys[path].Pool;
+    if (client.vgroups[path])
+        return path;
+    if (client.stratis_pools[path])
+        return path;
 }
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-19352-20231205-120952-f0d04c46-fedora-39-devel//Coverage/lcov/github-pr.diff.gcov.html)